src/main/java/org/edumips64/utils/cli/Cli.java

```diff
@@ -0,0 +1,128 @@
+package org.edumips64.utils.cli;
+
+import org.edumips64.core.CPU;
+import org.edumips64.core.Dinero;
+import org.edumips64.core.IOManager;
+import org.edumips64.core.Memory;
+import org.edumips64.core.SymbolTable;
+import org.edumips64.core.is.BUBBLE;
+import org.edumips64.core.is.InstructionBuilder;
+import org.edumips64.core.parser.Parser;
+import org.edumips64.utils.ConfigStore;
+import org.edumips64.utils.CurrentLocale;
+import org.edumips64.utils.MetaInfo;
+import org.edumips64.utils.io.LocalFileUtils;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+@Command(name="", resourceBundle = "CliMessages", subcommands = {
+        StepCommand.class,
+        RunCommand.class,
+        FileCommand.class,
+        ShowCommand.class,
+        DineroCommand.class,
+        ConfigCommand.class
+})
+public class Cli implements Runnable {
+
+    private Memory memory;
+    private CPU c;
+    private SymbolTable symTab;
+    private IOManager iom;
+    private Dinero dinero;
+    private InstructionBuilder instructionBuilder;
+    private Parser p;
+    private final ConfigStore configStore;
+
+    public Cli(ConfigStore cfg) {
+        this.configStore = cfg;
+        memory = new Memory();
```

Comment:
Could you just invoke `reset()` to do the initialization here, right after initializing `this.configStore`?

Reply:
Yeah, that's a good call. Done.